bool isLookup
---
int32 CUR_IQ_SETTING 
int32 CUR_PROPORTIONAL 
int32 CUR_INTEGRAL      
int32 CUR_ID_SETTING    
int32 CUR_MAXSPEED      
int32 ACTUAL_CURRENT    
int32 VEL_SETTING       
int32 VEL_PROPORTIONAL  
int32 VEL_INTEGRAL      
int32 VEL_OUTPUT_LIMITATION_MINIMUM  
int32 VEL_OUTPUT_LIMITATION_MAXIMUM  
int32 ACTUAL_VELOCITY 
int32 POS_SETTING  
int32 POS_PROPORTIONAL 
int32 POS_INTEGRAL 
int32 POS_DIFFERENTIAL 
int32 POS_OUTPUT_LIMITATION_MINIMUM 
int32 POS_OUTPUT_LIMITATION_MAXIMUM 
int32 POS_LIMITATION_MINIMUM 
int32 POS_LIMITATION_MAXIMUM 
int32 HOMING_POSITION 
int32 ACTUAL_POSITION 
int32 PROFILE_POS_MAX_SPEED 
int32 PROFILE_POS_ACC 
int32 PROFILE_POS_DEC 
int32 PROFILE_VEL_MAX_SPEED 
int32 PROFILE_VEL_ACC 
int32 PROFILE_VEL_DEC 
int32 POS_OFFSET 
int32 VOLTAGE 
int32 POS_LIMITATION_SWITCH 
int32 CURRENT_SCALE  
int32 VELOCITY_SCALE 
int32 FILTER_C_STATUS 
int32 FILTER_C_VALUE  
int32 FILTER_V_STATUS 
int32 FILTER_V_VALUE  
int32 FILTER_P_STATUS 
int32 FILTER_P_VALUE  
int32 LOCK_ENERGY 
int32 ACTUATOR_TEMPERATURE 
int32 INVERTER_TEMPERATURE 
int32 ACTUATOR_PROTECT_TEMPERATURE   
int32 ACTUATOR_RECOVERY_TEMPERATURE  
int32 INVERTER_PROTECT_TEMPERATURE   
int32 INVERTER_RECOVERY_TEMPERATURE  
int32 ACTUATOR_SWITCH  
int32 FIRMWARE_VERSION 
int32 ONLINE_STATUS 
int32 SN_ID    
int32 MODE_ID  
int32 ERROR_ID 
int32 CURRENT_LIMIT  
int32 VELOCITY_LIMIT 
int32 INIT_STATE  